File: beginner/javascript/1072.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split('\n');

var i, n, x, contIn = 0, contOut = 0;

n = parseInt(valores.shift());

for(i = 0; i < n; i++){
    x = parseInt(valores.shift());

    if(x >= 10 && x <= 20){
        contIn++;
    } else{
        contOut++;
    }
}

console.log(contIn + " in");
console.log(contOut + " out");

File: beginner/python/1072.py
contIn = 0
contOut = 0
i = 0

n = int(input())

for i in range(i,n):
    x = int(input())

    if(x >= 10 and x <= 20):
        contIn+=1
    else:
        contOut+=1
    

print("%d in" % contIn)
print("%d out" % contOut)